bonsai/sdk/Cargo.toml

```diff
@@ -30,6 +31,7 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 uuid = { version = "1.8", features = ["v4"] }
 
 [features]
-default = ["std"]
+default = ["std", "sync_client"]
 std = ["risc0-groth16/std"]
-async = ["dep:tokio"]
+sync_client = ["maybe-async/is_sync"]
+async_client = ["dep:tokio", "reqwest/stream"]
```

Comment:
What if a user does:

```
features = ["sync_client", "async_client"]
```

Does this even make sense?

Reply:
This is explicitly a know problem of maybe_async and not supported: https://github.com/fMeow/maybe-async-rs/issues/6